core/src/main/java/org/apache/iceberg/rest/responses/ConfigResponseParser.java

```diff
@@ -83,6 +89,19 @@ public static ConfigResponse fromJson(JsonNode json) {
               .collect(Collectors.toList()));
     }
 
+    if (json.hasNonNull(IDEMPOTENCY_KEY_LIFETIME)) {
+      String lifetime = JsonUtil.getString(IDEMPOTENCY_KEY_LIFETIME, json);
+      // empty or invalid -> treated as not advertised (skip setting)
+      if (lifetime != null && !lifetime.trim().isEmpty()) {
+        try {
+          Duration.parse(lifetime);
+          builder.withIdempotencyKeyLifetime(lifetime);
+        } catch (RuntimeException e) {
+          // ignore invalid value
```

Comment:
I don't think we should ignore this if the lifetime is an invalid value. Also I would add the parsing to `JsonUtil` to have `getDurationStringOrNull`(similar to `getStringOrNull`). That method would throw if the lifetime string can't be parsed to a string and would throw a `Cannot parse to a duration string value: ...`